### DISCUSSION TITLE
GitHub Billing Ticket 10 days no answer

### DISCUSSION THREAD

**Discussion by AleksanderGPL** [community/community#38929] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hey has anyone been experiencing super long wait times for ticket response? I have a very important ticket about billing, 10 days, no answer.

**Comment by EXeduardo** [comment]
same here!; I can't understant why github doesn't have a chat for billing or an phone number, I need to set up github as provider on my organization and just guthub dosen't help me with documentation, I will have to change to another platform just for this

**Comment by lukeryberg** [comment]
Im having the same issue, got a huge billing problem with copilot, and haven't heard anything, and is causing a huge stall in my project.

**Comment by kev-out** [comment]
My issue is of an Administrative nature but unfortunatly I am too waiting for a reply for over 2 weeks now  :( 
@github-staff  any update from your side for the little guy?

**Reply by pentestw0rkerbit0** [reply]
Same here no answer me back. Have issue with monthly to yearly upgrade.
